pxtblocks/blocklylayout.ts

```diff
@@ -338,9 +341,10 @@ namespace pxt.blocks.layout {
 
         if (!BrowserUtils.isEdge()) return Promise.resolve();
 
-        const images = xsg.getElementsByTagName("image") as NodeListOf<Element>;
+        const images = xsg.getElementsByTagName("image")
         const p = pxt.Util.toArray(images)
             .filter(image => /^data:image\/svg\+xml/.test(image.getAttributeNS(XLINK_NAMESPACE, "href")))
+            .map(img => img as unknown as HTMLImageElement)
```

Comment:
should be able to bump this up to the the first part to avoid a runtime cost just to make the type system happy `(pxt.Util.toArray(images) as HTMLImageElement[])`